src/main/java/org/codehaus/mojo/versions/PluginUpdatesXmlRenderer.java

```diff
@@ -106,11 +106,9 @@ public void render()
 
     private static String getSummaryBlock( Map<Plugin, PluginUpdatesDetails> allUpdates )
     {
-        Collection<ArtifactVersions> allVersions = new ArrayList<ArtifactVersions>();
-        for ( PluginUpdatesDetails details : allUpdates.values() )
-        {
-            allVersions.add( details.getArtifactVersions() );
-        }
+        Collection<ArtifactVersions> allVersions = allUpdates.values().stream()
+            .map( details -> details.getArtifactVersions() )
+            .collect( Collectors.toList() );
```

Comment:
I won't mind a static import here, but however you like it better.

Reply:
personnal choice I don't like